sdk/storage/Azure.Storage.Files.Shares/tests/FileTestBase.cs

```diff
@@ -113,6 +114,15 @@ public ShareServiceClient GetServiceClient_SharedKey()
                         TestConfigDefault.AccountKey),
                     GetOptions()));
 
+        public ShareServiceClient GetServiceClient_OAuth()
```

Comment:
Should this be GetServiceClient_OAuth_SharedKey like in blobs? otherwise it may give illusion of oauth support in shares.

Reply:
fixed.